.travis.yml

```diff
@@ -48,10 +48,10 @@ jobs:
         - make -j2
         - ctest -V
 
-    - name: macOS 10.15, llvm 10
+    - name: macOS 10.15, llvm 11
```

Comment:
Since we are installing the latest llvm version, what about changing `llvm 11` with something like "latest llvm"?

Reply:
Maybe `lastest llvm (version 11)`?